### ISSUE TITLE
:fulcro.client.primitives/component-or-reconciler spec looks wrong

### ISSUE THREAD

**Issue by OscarMarshall** [fulcrologic/fulcro#189]
The spec [here](https://github.com/fulcrologic/fulcro/blob/305eb8f864d790cd2905e4ec7ce349d4dbfb2ae0/src/main/fulcro/client/primitives.cljc#L99) looks wrong.

Instead of:

```clojure
(s/def ::component-or-reconciler vector?)
```

I believe it should be:

```clojure
(s/def ::component-or-reconciler (s/or :component component?, :reconciler reconciler?))
```

Or I have grossly misunderstood what this spec is intended to be used for. It looks like it's not actually used anywhere, though, so I'm guessing it just slipped through the cracks.

**Comment by awkay** [comment]
I would agree that it is wrong...not sure how that ended up in there...probably copy-pasted something, got interrupted, and forgot where I was.
